lib/cc/cli/init.rb

```diff
@@ -7,6 +7,12 @@ module CLI
     class Init < Command
       include CC::Analyzer
 
+      ARGUMENT_LIST = "[--upgrade]".freeze
+      SHORT_HELP = "Generate a configuration and install the right images based on the contents of your repo.".freeze
```

Comment:
📖  "Generate a configuration based on the contents of your repo."

We can leave out the image pulling for this